resources/guides/ta_guide.md

```diff
@@ -0,0 +1,99 @@
+---
+id: teaching assistant
+title: How To Become a Teaching Assistant
+sidebar_label: How To Become a Teaching Assistant
+slug: /guides/ta_guide
```

Comment:
Would be nice for the slug to be more closely related to the page title/label. Perhaps something like `/guides/becoming_a_ta`?

Related:
@BorhanSaflo I was looking through our codebase and noticed that we have a bunch of different slug formats:
- camelCase
- alllowercase
- spaces_become_underscores
- spaces-become-hyphens

Is there a preferred one? And if not, we should choose one and document it somewhere.

Reply:
https://github.com/uwindsorcss/wiki/issues/163
look to this issue, we should make the slugs more consistent

Reply:
I see hyphenated slugs are preferred as of now, I will update to use that format